### PR TITLE
Add cli for dpc-go migrations

### PR DESCRIPTION
**Pull request by SalGutierrez**
### Fixes [DPC-1137](https://jira.cms.gov/browse/DPC-1137)

We need a way to do schema migrations in the new DPC-GO project.
### Proposed Changes

Use the golang-migrations library

https://github.com/golang-migrate/migrate
### Change Details

- added docker file for running the migration library.
- added migration to create org table.
- added a README for creating and running migrations.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### CODE REVIEW

**Comment by whuang85** [comment]
Any reason to start the migration file name at 10001?

File: dpc-go/dpc-attribution/migrator/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+####Running Migrations
+1) Build the docker image (uses the `Dockerfile` found in this directory) that will be used to run the migration commands.
+`docker build -t migrator .`
+
+*Note: The `Dockerfile` copies the files under /migrations to the image's own volume.*
+
+2) Run the migrations: 
+ `docker run  -network host migrator -database "postgresql://postgres:dpc-safe@localhost"  up`
```
Comment (dhgreene) [inline dpc-go/dpc-attribution/migrator/README.md:28]: Is it worth creating a Makefile target for this such as `make migrate` that's easier to remember?

**Review by dhgreene — COMMENTED** [review]
Should we have a `10001.create_org_table.down.sql` migration, if for no other reason than as a model the next person?

**Comment by SalGutierrez** [comment]
> Any reason to start the migration file name at 10001?

I'm sure I had a reason a few days ago, but I kind of forgot.

We start at 1.

**Comment by SalGutierrez** [comment]
> Any reason to start the migration file name at 10001?

Changed to 1

**Comment by SalGutierrez** [comment]
> Should we have a `10001.create_org_table.down.sql` migration, if for no other reason than as a model the next person?

Added.

File: dpc-go/dpc-attribution/migrator/README.md (hunk 1 above)
Comment (SalGutierrez) [reply dpc-go/dpc-attribution/migrator/README.md:28]: we could, would that be part of this ticket or a new one?

**Review by whuang85 — APPROVED** [review]

File: dpc-go/dpc-attribution/migrator/README.md (hunk 1 above)
Comment (dhgreene) [reply dpc-go/dpc-attribution/migrator/README.md:28]: That can go in a new ticket.

**Review by dhgreene — APPROVED** [review]
🚗

**Pull request merged**